cmake/OmimHelpers.cmake

```diff
@@ -79,14 +79,26 @@ function(omim_add_library library)
 
   # Enable warnings for all our libraries.
   target_compile_options(${library} PRIVATE ${OMIM_WARNING_FLAGS})
-  target_include_directories(${library} PRIVATE ${OMIM_INCLUDE_DIRS})
+  target_include_directories(${library} SYSTEM PRIVATE ${OMIM_INCLUDE_DIRS})
   if (USE_PPROF AND PLATFORM_MAC)
     find_path(PPROF_INCLUDE_DIR NAMES gperftools/profiler.h)
     target_include_directories(${library} SYSTEM PUBLIC ${PPROF_INCLUDE_DIR})
   endif()
   if (USE_PCH)
     add_precompiled_headers_to_target(${library} ${OMIM_PCH_TARGET_NAME})
   endif()
+
+
+  if (USE_CLANG_TIDY)
```

Comment:
Does it make sense to avoid adding another option and leverage the CXX_CLANG_TIDY that is set from the command line by `-DCXX_CLANG_TIDY=...`?

Reply:
I don't think so. The problem is that we have to set it to one targets, but don't want to set it to other ones. We probably could do smth like this with `CMAKE_CXX_CLANG_TIDY`, but in this case clang tidy applies for the whole project, i.e. if in some dependency we have issue(it can be something trivial like I fixed here - e.g. missed `reserve`) - we won't be able to build the project. WDYT?

Reply:
Agree.